google/cloud/aiplatform/lro.py

```diff
@@ -1,109 +0,0 @@
-# -*- coding: utf-8 -*-
-
-# Copyright 2020 Google LLC
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-#
-
-import proto
-
-from google.api_core import operation as ga_operation
-from google.cloud.aiplatform import base
-from typing import Optional, Callable
-
-
-class LRO:
-    """A handler for operation futures."""
```

Comment:
Is the class being removed to lack of use?

Reply:
Yes, it's currently unused throughout the SDK. We can restore it if needed.